### ISSUE TITLE
remotesensing_dockwidget: year 58668 is out of range

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#383]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/383) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

```
#!python

______________________ Test_MSSTopViewWindow.test_open_rs ______________________

self = <test_topview.Test_MSSTopViewWindow object at 0x7f4a6d9a75c0>
mockcrit = <MagicMock name='QMessageBox' id='139957356757736'>

    @mock.patch("mslib.msui.mss_qt.QtWidgets.QMessageBox")
    def test_open_rs(self, mockcrit):
        self.window.cbTools.currentIndexChanged.emit(3)
        QtWidgets.QApplication.processEvents()
>       rsdock = self.window.docks[2].widget()
E       AttributeError: 'NoneType' object has no attribute 'widget'

mslib/msui/_tests/test_topview.py:105: AttributeError
----------------------------- Captured stderr call -----------------------------
[#################################] 100% de421.bsp
[#################################] 100% deltat.data
[#################################] 100% deltat.preds
Traceback (most recent call last):


  File "/home/user/PycharmProjects/mss/mslib/msui/topview.py", line 225, in openTool
    widget = rs.RemoteSensingControlWidget(view=self.mpl.canvas)
  File "/home/user/PycharmProjects/mss/mslib/msui/remotesensing_dockwidget.py", line 60, in __init__
    self.timescale = self.load.timescale()
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 270, in timescale
    preds = self('deltat.preds')
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 184, in __call__
    expiration_date, data = parser(f)
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 347, in parse_deltat_preds
    expiration_date = date(year[0] + 2, month[0], 1)
ValueError: year 58668 is out of range
CRITICAL:root:MSS Version: 1.7.5
CRITICAL:root:Python Version: 3.7.2 (default, Dec 29 2018, 06:19:36) 
[GCC 7.3.0]
CRITICAL:root:Platform: Linux-4.4.0-142-generic-x86_64-with-debian-stretch-sid (('64bit', ''))
CRITICAL:root:Fatal error: Traceback (most recent call last):
  File "/home/user/PycharmProjects/mss/mslib/msui/topview.py", line 225, in openTool
    widget = rs.RemoteSensingControlWidget(view=self.mpl.canvas)
  File "/home/user/PycharmProjects/mss/mslib/msui/remotesensing_dockwidget.py", line 60, in __init__
    self.timescale = self.load.timescale()
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 270, in timescale
    preds = self('deltat.preds')
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 184, in __call__
    expiration_date, data = parser(f)
  File "/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py", line 347, in parse_deltat_preds
    expiration_date = date(year[0] + 2, month[0], 1)
ValueError: year 58668 is out of range
```

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

This is caused by a "bug" in skyfield, which has been fixed in version 1.10 for over four weeks, also on conda-forge.
https://github.com/skyfielders/python-skyfield/pull/236
As such, an update of your packages should fix the problem...

The good thing is, that the bug will only express itself when using the remote sensing widget.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

great than we have only to fixate the version in meta.yaml

also on condaforge.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

conda-forge build triggerd,

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

no news yet on

https://github.com/conda-forge/mss-feedstock/pull/44

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

against which version shall the version number be patched? stable/develop?

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Fixed skyfield version to one fixing issue 383

Fixes issue #383

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Fixed skyfield version to one fixing issue 383

Fixes issue #383

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in joernu76/mss/issue383 (pull request #608)

Fixed skyfield version to one fixing issue 383
